airbyte-integrations/connectors/source-monday/unit_tests/test_graphql_request_options_provider.py

```diff
@@ -0,0 +1,29 @@
+from source_monday import GraphQLRequestOptionsProvider
+
+
+def test_get_request_params(mocker):
+    schema = {
```

Comment:
I suggest you move it to parametrize(), it would enable us extend the test in the future